assets/javascripts/comments.js

```diff
@@ -51,15 +47,15 @@ function deleteComment(deleteButton) {
   if (!window.confirm('Do you really want to delete the comment written by ' + author + '?')) {
     return;
   }
-  $.ajax({
-    url: deleteButton.dataset.deleteUrl,
-    method: 'DELETE',
-    success: () => {
+  fetchWithCSRF(deleteButton.dataset.deleteUrl, {method: 'DELETE'})
+    .then(response => {
+      if (response.status != 200) throw `Server returned ${response.status}: ${response.statusText}`;
```

Comment:
Why `response.status` is limited on 200. I wonder if `response.ok` is more appropiate in those if https://developer.mozilla.org/en-US/docs/Web/API/Response/ok